docs/blog/posts/2023-07-17-no-unlimited-free-usage.md

```diff
@@ -0,0 +1,65 @@
+---
+draft: false
+date: 2023-07-17
+search:
+  exclude: true
+authors:
+  - fkorotkov
+categories:
+  - announcement
+---
+
+# Limiting free usage of Cirrus CI
+
+Unfortunately the day has come. As a self-bootstrapped company Cirrus Labs can no longer provide unlimited usage of Cirrus CI
+for public repositories for free. We have to put a limit on the amount of compute resources that can be consumed for free each month
+by organizations and users. Starting September 1st 2023, there will be an upper monthly limit on free usage equal to 40 compute credits
+(equal to 10,000 CPU-minutes for Linux tasks).
+
+The reason for the change is that we want to continue being a profitable business and keep Cirrus CI running,
+but unfortunately we haven’t found a better solution for a couple of ongoing issues described below.
+
+<!-- more -->
+
+**Crypto miners are still active.** Methodically, a lot of CI vendors one after another were restricting free usage because of this single reason.
+Only Cirrus CI and GitHub Actions have been allowing unlimited usage as of lately and we are very proud of the effort that we have put into battling with the abuse.
+We tried everything from clever firewall and traffic analysis to some basic machine learning on factors like similarity of config files and CPU usage patterns.
+This effort got us a silver medal in the race of providing unlimited free usage for as long as we could. Congrats GitHub Actions on getting the gold
+and remaining the only CI with free unlimited usage for public repositories.
+
+**Cirrus CI usage pattern in many cases is not optimal.** This is just an observation we made during the decision-making process.
+It appears that free Cirrus CI tasks **have only 30-40% CPU utilization on average**. On the other hand,
+paid **tasks that use compute credits have an average CPU utilization of 80%**. We randomly picked a handful of tasks with low CPU utilization
+and discovered that many people just used the maximum possible resources that Cirrus CI allows “just because they could”.
+Frequently we saw tasks requesting 8 CPUs and 24GB of memory, but in reality they used only a single CPU core.
+
+In addition to introducing the limits we will also lower the prices for the existing compute resources. Starting August 1st,
```

Comment:
What do you think about making this a section (`# Good news`, `# Cheaper compute time`, or something like that)?

Otherwise it's easy to miss this piece, but it sends a pretty important message too.

Reply:
See [0e2a0b8](https://github.com/cirruslabs/cirrus-ci-docs/pull/1197/commits/0e2a0b8bf6d944f12df519a91ac8e9f03c392bb5)